jax/lax/lax_parallel.py

```diff
@@ -38,7 +38,7 @@
 
 ### parallel traceables
 
-def psum(x, axis_name):
+def psum(x, axis_name, axis_index_groups=None):
```

Comment:
I also kind of like just `groups`, though verbosity seems less bad for nonobvious/"power user" features than it would be for something more central to the API. Matt suggested we start with an underscored name; WDYT about that option?
(I think my very weak overall preference is still `axis_index_groups`.)

Reply:
Starting with an underscored name sounds fine to me, although this API seems sane enough to me that I don't think we need to be so defensive. My slight preference is no underscore, to avoid having an underscored param forever that everyone eventually uses anyway, but either is good with me.

> verbosity seems less bad for nonobvious/"power user" features
I agree, I just don't think the extra verbosity helps describe what this is. Or maybe it does? I can no longer see it with fresh eyes :) `axis_index_groups` is fine, it at least gives us a very specific name to call these things.

Reply:
Yeah I agree this is looking solid enough that we might not need to be defensive.

On the name, I like James's idea!